content/kubermatic/master/architecture/support_policy/provider_support_matrix/kubevirt/kubevirt.en.md

```diff
@@ -21,6 +21,32 @@ KKP supports KubeVirt Operator >= 0.19.0 and the Containerized Data Importer >=
 to run KubeVirt, however a Kubernetes cluster consists of 3 nodes with 2 CPUs, 4GB of RAM and 30GB of storage, to have a
 minimal installation.
 
+### KubeVirt Configuration Requirements
+Kubevirt requires the following configuration to be used with KKP.
+- In case your Kubevirt namespace has the ConfigMap 'kubevirt-config' then use this ComfigMap for adding the feature gates to it. Look at the path `{.data.feature-gates}`
```

Comment:
Small typo, instead `Comfigmap` it should be `ConfigMap`

Reply:
Fixed.